wails-frontend/src/pages/PasswordPromptHandler/Default.tsx

```diff
@@ -56,6 +57,9 @@ export function Default() {
       if (CodeMessage === ErrorCode.WrongPassword) {
         setError({ password: Intl.t(`errors.${CodeMessage}`) });
         return;
+      } else {
+        setErrorMessage(Intl.t(`errors.default`));
+        return;
       }
```

Comment:
you chose ;) I prefer the first one bellow:

```
      if (CodeMessage === ErrorCode.WrongPassword) {
        setError({ password: Intl.t(`errors.${CodeMessage}`) });
      } else {
        setErrorMessage(Intl.t(`errors.default`));
      }
      return
```

```
      if (CodeMessage === ErrorCode.WrongPassword) {
        setError({ password: Intl.t(`errors.${CodeMessage}`) });
        return
      }
      setErrorMessage(Intl.t(`errors.default`));
      return
```

Reply:
the first one it is !